tests/lib/rules/newline-after-var.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @fileoverview Tests for newline-after-var rule.
+ * @author Gopal Venkatesan <http://g13n.me>
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var eslint = require("../../../lib/eslint"),
+    ESLintTester = require("eslint-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+var eslintTester = new ESLintTester(eslint);
+
+eslintTester.addRuleTest("lib/rules/newline-after-var", {
+    valid: [
+        { code: "var greet = 'hello'; console.log(greet);", args: [2, "never"] },
```

Comment:
I know. `args` is deprecated in tests. I'm asking you to replace with `options` that does the same thing but doesn't require the `2` (which is unused in tests).